install_bootstrap

```diff
@@ -282,6 +282,9 @@ while [[ "${run_boardwalk^^}" != 'Y' &&  "${run_boardwalk^^}" != 'N' ]] ; do
         ############################
         # AWS questions            #
         ############################
+        aws_default_region='aws_default_region'
+        ask_question "What is your AWS Default Region?" "$AWS_DEFAULT_REGION" "AWS Default Region" $aws_default_region
```

Comment:
I think you should get rid of the word `Default` from the prompt. Ideally you'd rename the property too in here and all the files, but we can leave it as is unless you're inclined to change it.

Reply:
I agree it's clearer to remove it from the install script prompt, but I'd like to keep it in the variable b/c that lines up with that the actual variable is called for boto.